loadflow-validation/src/main/java/eu/itesla_project/loadflow/validation/CheckFlowsTool.java

```diff
@@ -0,0 +1,134 @@
+
+        @Override
+        public String getName() {
+            return "check-flows";
+        }
+
+        @Override
+        public String getTheme() {
+            return "Loadflow Validation";
+        }
+
+        @Override
+        public String getDescription() {
+            return "Check flows of a network";
+        }
+
+        @Override
+        public Options getOptions() {
+            Options options = new Options();
+            options.addOption(Option.builder().longOpt("case-file")
+                    .desc("case file path")
+                    .hasArg()
+                    .argName("FILE")
+                    .required()
+                    .build());
+            options.addOption(Option.builder().longOpt("output-file")
+                    .desc("output file path")
+                    .hasArg()
+                    .argName("FILE")
+                    .required()
+                    .build());
+            options.addOption(Option.builder().longOpt("load-flow")
+                    .desc("run loadflow")
+                    .build());
+            options.addOption(Option.builder().longOpt("verbose")
+                    .desc("verbose output (true/false)")
+                    .hasArg()
+                    .argName("TRUE/FALSE")
+                    .build());
+            return options;
+        }
+
+        @Override
+        public String getUsageFooter() {
+            return null;
+        }
+
+    };
+
+    @Override
+    public Command getCommand() {
+        return COMMAND;
+    }
+
+    @Override
+    public void run(CommandLine line) throws Exception {
+        Path caseFile = Paths.get(line.getOptionValue("case-file"));
+        Path outputFile = Paths.get(line.getOptionValue("output-file"));
+        CheckFlowsConfig config = CheckFlowsConfig.load();
+        if (line.hasOption("verbose")) {
+            config.setVerbose(Boolean.parseBoolean(line.getOptionValue("verbose")));
+        }
+        if (Files.isRegularFile(caseFile)) {
+            System.out.println("Loading case " + caseFile);
+            Network network = Importers.loadNetwork(caseFile);
+            if (network == null) {
+                throw new RuntimeException("Case " + caseFile + " not found");
+            }
+            if (line.hasOption("load-flow")) {
+                System.out.println("Running loadflow on network " + network.getId());
+                try (ComputationManager computationManager = new LocalComputationManager()) {
+                    LoadFlowParameters parameters = new LoadFlowParameters().setVoltageInitMode(LoadFlowParameters.VoltageInitMode.UNIFORM_VALUES)
+                                                                            .setTransformerVoltageControlOn(false)
+                                                                            .setNoGeneratorReactiveLimits(false)
+                                                                            .setPhaseShifterRegulationOn(false);
```

Comment:
The LoadFlow parameters should be read from the configuration file (load-flow-default-parameters)